addons/account/views/account_journal_views.xml

```diff
@@ -56,38 +55,29 @@
                                 <group>
                                     <group string="Accounting Information">
                                         <field name="default_account_type" invisible="1"/>
-                                        <!-- Bank -->
-                                        <field name="default_account_id" string="Bank Account"
-                                               attrs="{'required': [('id', '!=', False), ('type', '=', 'bank')], 'invisible': [('type', '!=', 'bank')]}"
+                                        <label for="default_account_id" string="Bank Account"
+                                               attrs="{'invisible': [('type', '!=', 'bank')]}" groups="account.group_account_readonly"/>
+                                        <label for="default_account_id" string="Cash Account"
+                                               attrs="{'invisible': [('type', '!=', 'cash')]}" groups="account.group_account_readonly"/>
+                                        <label for="default_account_id" string="Default Income Account"
+                                               attrs="{'invisible': [('type', '!=', 'sale')]}" groups="account.group_account_readonly"/>
+                                        <label for="default_account_id" string="Default Expense Account"
+                                               attrs="{'invisible': [('type', '!=', 'purchase')]}" groups="account.group_account_readonly"/>
+                                        <label for="default_account_id" string="Default Account"
+                                               help="If set, this account is used to automatically balance entries."
```

Comment:
Note that the help is not displayed, and wasn't displayed before the fix either

Reply:
It was displayed before to me :thinking: But indeed it doesn't work like this, I don't know if it's an issue for the PO